RecoEcal/Configuration/python/RecoEcal_EventContent_cff.py

```diff
@@ -66,7 +64,7 @@
 )
 RecoEcalRECO.outputCommands.extend(RecoEcalAOD.outputCommands)
 _phase2_hgcal_scCommands = ['keep *_particleFlowSuperClusterHGCal_*_*',
-                            'keep *_particleFlowSuperClusterHGCalFromMultiCl_*_*']
+                            'keep *_particleFlowSuperClusterHGCal_*_*']
```

Comment:
```suggestion
                            ]
```
this is a duplicate now (please edit to have just one line)

Reply:
Done.